build_wasm.sh

```diff
@@ -20,18 +17,21 @@ FUNCTIONS='['
 FUNCTIONS+='_dedx_get_program_list,'
 FUNCTIONS+='_dedx_get_material_list,'
 FUNCTIONS+='_dedx_get_ion_list,'
-FUNCTIONS+='_dedx_get_min_energy,'
-FUNCTIONS+='_dedx_get_max_energy,'
 FUNCTIONS+='_dedx_get_ion_name,'
 FUNCTIONS+='_dedx_get_material_name,'
 FUNCTIONS+='_dedx_get_program_name,'
+FUNCTIONS+='_dedx_get_min_energy,'
+FUNCTIONS+='_dedx_get_max_energy,'
+FUNCTIONS+='_dedx_get_simple_stp,'
 FUNCTIONS+='_malloc,'
 FUNCTIONS+='_free'
 
-
 FUNCTIONS+=']'
 
-emcc libdedx.a -o $PROJECT_NAME.js -s EXPORTED_FUNCTIONS="$FUNCTIONS" -s ENVIRONMENT='web' -s USE_ES6_IMPORT_META=0 -s EXPORT_ES6=1 -s MODULARIZE=1 -s WASM=1 -s EXPORTED_RUNTIME_METHODS=["ccall","cwrap","UTF8ToString"]
+# remember to use --embed-file instead of --preload-file for web based development
```

Comment:
explain why we should use --embed-file

Reply:
Embed- as opposed to preload - embeds the data into the output .js file. They both should have the same effect - the data should be available on the site but the preload option requires the generated .js file to load the file when it sets the module up. This wasn't working correctly - it seemed to have some problems fetching the file, either caused by CORS or some other reason. On the other hand the embed option worked (cause i can see below it doesn't anymore) flawlesly without messing with the generated code anymore. Yes, it is somewhat less efficient - the data is stored and fetched as text (whereas with preload it should be fetched as binary) but the size of our dataset isn't staggering so it doesn't cause much of a slowdown. In short it's more comfortable and reliable when using the generated file as a react module.

Reply:
```suggestion
# remember to use --embed-file instead of --preload-file for web based development
# # Embed- as opposed to preload - embeds the data into the output .js file. 
# They both should have the same effect - the data should be available on the site 
# but the preload option requires the generated .js file to load the file when it sets the module up. 
# This wasn't working correctly - it seemed to have some problems fetching the file, 
# either caused by CORS or some other reason. 
# On the other hand, the embed option worked flawlessly without messing with the generated code anymore. 
# Yes, it is somewhat less efficient - the data is stored and fetched as text 
# (whereas with preload it should be fetched as binary) 
# but the size of our dataset isn't staggering 
# so it doesn't cause much of a slowdown. 
# In short it's more comfortable and reliable when using the generated file as a react module.
```

Reply:
Better to add that as a comment in the code